lib/services.template.ejs

```diff
@@ -7,13 +7,13 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
   //   import lbServices from './lb-services';
   //   angular.module('app', [lbServices]);
   //
-  module.exports = <%-: moduleName | q %>;
+  module.exports = <%-: helpers.quotedString(moduleName) %>;
```

Comment:
I think you need to replace all occurences of `<%-:` with `<%-`. IIRC, the `:` suffix was enabling filters via the `|` operator. AFAICT, it's no longer available in the latest EJS version (see e.g. [here](https://www.npmjs.com/package/ejs#tags)).

Could you please try to make this change and see if it fixes the problem you are observing?